### PR TITLE
Release 2.3 - This time for real

### PR DESCRIPTION
**Pull request by WackerO**
Second release PR after I found a bug in the previous one

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/qbic-pipelines/rnadeseq/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,11 +15,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- [#243](https://github.com/qbic-pipelines/rnadeseq/pull/243) Release 2.3 to master, part II (found a bug before actual release)
```
Comment (skrakau) [inline CHANGELOG.md:18]:
```suggestion
- [#244](https://github.com/qbic-pipelines/rnadeseq/pull/244) Release 2.3 to master, part II (found a bug before actual release)
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -193,6 +193,16 @@
                     "type": "string",
                     "default": "${projectDir}/assets/references.bibtex",
                     "description": "Reference bibtex file for the report, should not be changed."
+                },
```
Comment (LaurenceKuhl) [inline nextflow_schema.json:196]: why is it a variable if it shouldn't be changed?

**Review by LaurenceKuhl — APPROVED** [review]

**Review by skrakau — APPROVED** [review]
Apart from the typo it looks good to me!

File: nextflow_schema.json (hunk 2 above)
Comment (WackerO) [reply nextflow_schema.json:196]: There was apparently some nextflow change; now it seems the files in assets/ cannot be accessed by processes unless they are provided to the respective module as channels.

**Pull request merged**